composer.json

```diff
@@ -161,6 +161,9 @@
             "drupal/acquia_telemetry-acquia_telemetry": {
                 "Add check for core index": "https://www.drupal.org/files/issues/2020-08-18/3165473-27.patch"
             },
+            "drupal/config_ignore": {
```

Comment:
@chandan-singh7929 Please update composer.json file for acquia_cms_site_studio module as well and include this patch there as well.
@kdruckman Do we also need to pin the config_ignore module in composer.json as we've added the patch ?